### PR TITLE
New meta

### PR DESCRIPTION
**Pull request by azoyan**

### CODE REVIEW

**Review by bezrazli4n0 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,35 +1,40 @@
...
+	@echo ⚙️ Installing a toolchain \& a target...
 	@rustup toolchain add nightly
 	@rustup target add wasm32-unknown-unknown --toolchain nightly
 
-linter:
-	@echo ──────────── Run linter ───────────────────────
-	@cargo +nightly clippy --all-targets -- --no-deps -D warnings
+lint:
+	@echo ⚙️ Running the linter...
+	@cargo +nightly clippy -- -D warnings
```
Comment (fluiderson) [inline Makefile:25]:
```suggestion
	@cargo +nightly clippy --workspace -- -D warnings
```

File: io/src/lib.rs [hunk 2]
```diff
@@ -214,10 +226,26 @@ pub enum StateReply {
 }
 
 #[derive(Debug, Default, Clone, Encode, Decode, TypeInfo, PartialEq)]
+#[codec(crate = gstd::codec)]
+#[scale_info(crate = gstd::scale_info)]
```
Comment (fluiderson) [inline io/src/lib.rs:230]:
```suggestion
```
Unneeded since `io/Cargo.toml` contains the `scale-info` and `codec` deps.

File: io/src/lib.rs [hunk 3]
```diff
@@ -214,10 +226,26 @@ pub enum StateReply {
...
     pub bet_size: u128,
     pub players_count_limit: u8,
     pub entry_timeout_ms: u64,
     pub move_timeout_ms: u64,
     pub reveal_timeout_ms: u64,
 }
+
+#[derive(Debug, Default, Encode, Decode, TypeInfo)]
+#[codec(crate = gstd::codec)]
+#[scale_info(crate = gstd::scale_info)]
```
Comment (fluiderson) [inline io/src/lib.rs:241]:
```suggestion
```
See the previous comment.

File: src/contract.rs [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    title: "RockPaperScissors",
+    init:
+        input: GameConfig,
+    handle:
+        input: Action,
+        output: Event,
+    state:
+        input: State,
+        output: StateReply,
+}
```
Comment (fluiderson) [inline src/contract.rs:205]:
```suggestion
```
Old meta.

**Review by fluiderson — APPROVED** [review]

**Pull request merged**